main/menu.js

```diff
@@ -92,6 +92,30 @@ function openSubtitles () {
   }
 }
 
+function skipForward () {
+  if (windows.main) {
+    windows.main.send('dispatch', 'skip', 1)
+  }
+}
+
+function skipBack () {
+  if (windows.main) {
+    windows.main.send('dispatch', 'skip', -1)
+  }
+}
+
+function increasePlaybackRate () {
+  if (windows.main) {
+    windows.main.send('dispatch', 'setPlaybackRate', 1)
```

Comment:
Would call this `changePlaybackRate`, otherwise it looks like we're setting the playback rate to 1 or -1